src/features/user/Account/components/AccountRecord.tsx

```diff
@@ -36,11 +36,22 @@ export default function AccountRecord({
   isModal = false,
 }: Props): ReactElement {
   const { t } = useTranslation(['me']);
+  const [showCertificate, setShowCertificate] = React.useState<boolean>(false);
 
   const outerDivClasses = isModal
     ? styles.recordModal
     : `${styles.record} ${selectedRecord === index ? styles.selected : ''}`;
 
+  useMemo(() => {
+    if (
+      record?.details?.donorCertificate ||
+      record?.details?.taxDeductibleReceipt ||
+      record?.details?.giftCertificate
+    ) {
+      setShowCertificate(true);
+    }
+  }, [record]);
+
```

Comment:
@sunilsabatp You can use `useMemo` to calculate the value of `showCertificate` without using state. 

Something like

```
const showCertificate = useMemo(
() => {
   if (
      record?.details?.donorCertificate ||
      record?.details?.taxDeductibleReceipt ||
      record?.details?.giftCertificate
    ) {
      return true;
    }
   return false;
},
[record]
),
```